src/coreclr/vm/ecalllist.h

```diff
@@ -556,11 +556,14 @@ FCFuncStart(gRuntimeHelpers)
     FCFuncElement("TryEnsureSufficientExecutionStack", ReflectionInvocation::TryEnsureSufficientExecutionStack)
     FCFuncElement("AllocTailCallArgBuffer", TailCallHelp::AllocTailCallArgBuffer)
     FCFuncElement("GetTailCallInfo", TailCallHelp::GetTailCallInfo)
-    FCFuncElement("RegisterForGCReporting", GCReporting::Register)
-    FCFuncElement("UnregisterForGCReporting", GCReporting::Unregister)
     FCFuncElement("Box", JIT_Box)
 FCFuncEnd()
 
+FCFuncStart(gRuntimeImports)
+    FCFuncElement("RhRegisterForGCReporting", GCReporting::Register)
```

Comment:
We should avoid introducing the `Rh...` prefix `RuntimeImports` into CoreCLR. The naming conventions for FCalls documented in https://github.com/dotnet/runtime/blob/main/docs/design/coreclr/botr/corelib.md and we should follow those. 

I hope we will get rid of the `Rh...` prefix and the extra `RuntimeImports` layer in native AOT at some point.

Reply:
OK thanks - I removed the "Rh" from CoreCLR\Mono.